tajo-plan/src/main/java/org/apache/tajo/plan/util/PlannerUtil.java

```diff
@@ -133,11 +134,90 @@ public static boolean checkIfSimpleQuery(LogicalPlan plan) {
           }
         }
       }
+
+      if (!noWhere && scanNode.getTableDesc().isExternal() && scanNode.getTableDesc().getPartitionMethod() != null) {
```

Comment:
After fixing that issue, I planned to move to managed table creation. 
Actually, I don't know Tajo internal managed table file structure. 
I'm familiar with Hive only.